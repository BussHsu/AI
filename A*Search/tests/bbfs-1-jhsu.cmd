--energy 247 --start-x 2 --start-y 1 --end-x 4 --end-y 4
