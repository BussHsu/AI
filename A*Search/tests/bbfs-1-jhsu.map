10  5 96 95 90 85
15  4 97 98 99 80
20  3 22 33 88 75
25  2 11 44 77 70
30  1  0 55 66 65
35 40 45 50 55 60
