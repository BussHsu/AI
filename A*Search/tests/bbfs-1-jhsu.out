Solution steps: ['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E', 'E', 'E', 'E', 'N', 'N', 'N', 'N', 'N', 'W', 'W', 'W', 'S', 'E', 'E']
Solution cost: 486
Number of states considered: 32

Frontier:
Pos=(3, 0) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N', 'N', 'N', 'N', 'N', 'E', 'E'] Cost=242

Closed List:
Pos=(1, 3) Moves=['W', 'N', 'N'] Cost=6
Pos=(2, 1) Moves=[] Cost=0
Pos=(5, 1) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N', 'N', 'N', 'N'] Cost=164
Pos=(2, 5) Moves=['E', 'E', 'S'] Cost=6
Pos=(0, 3) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S'] Cost=88
Pos=(4, 0) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N', 'N', 'N', 'N', 'N', 'E'] Cost=216
Pos=(1, 2) Moves=['W', 'N'] Cost=4
Pos=(5, 5) Moves=['E', 'E', 'S', 'W', 'W', 'W'] Cost=60
Pos=(4, 4) Moves=[] Cost=0
Pos=(1, 5) Moves=['W', 'N', 'N', 'N', 'N'] Cost=10
Pos=(5, 0) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N', 'N', 'N', 'N', 'N'] Cost=190
Pos=(0, 4) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S'] Cost=62
Pos=(5, 3) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N', 'N'] Cost=112
Pos=(4, 1) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N', 'N', 'N', 'N', 'E'] Cost=166
Pos=(1, 1) Moves=['W'] Cost=2
Pos=(5, 4) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N'] Cost=86
Pos=(0, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S'] Cost=166
Pos=(4, 5) Moves=['E', 'E', 'S', 'W', 'W'] Cost=34
Pos=(2, 2) Moves=['W', 'N', 'E'] Cost=86
Pos=(1, 4) Moves=['W', 'N', 'N', 'N'] Cost=8
Pos=(2, 3) Moves=['W', 'N', 'E', 'N'] Cost=208
Pos=(4, 2) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N', 'N', 'N', 'E'] Cost=146
Pos=(1, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E'] Cost=192
Pos=(3, 5) Moves=['E', 'E', 'S', 'W'] Cost=8
Pos=(0, 1) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S'] Cost=140
Pos=(5, 2) Moves=['E', 'E', 'S', 'W', 'W', 'W', 'N', 'N', 'N'] Cost=138
Pos=(0, 2) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S'] Cost=114
Pos=(2, 0) Moves=['W', 'N', 'N', 'N', 'N', 'W', 'S', 'S', 'S', 'S', 'S', 'E', 'E'] Cost=218
Pos=(4, 3) Moves=['N'] Cost=122
Pos=(0, 5) Moves=['W', 'N', 'N', 'N', 'N', 'W'] Cost=36
Pos=(3, 4) Moves=['E'] Cost=2
Pos=(2, 4) Moves=['E', 'E'] Cost=4
